File: erl2c/erl2ocaml_examples/example02.erl
-module(example02).
-compile(export_all).

odd(X) ->
  even(X).

id(X) ->
  X.

id_caller(X) ->
  id(X).

id_rec(X) ->
  id_rec(X).

even(X) ->
  odd(X).

File: erl2c/erl2ocaml_examples/example02.ml
let same''2 (x, y) = x == y

let rec odd'1 = function v_X -> even'1 v_X

and even'1 = function v_X -> odd'1 v_X

let rec id'1 = function v_X -> v_X

let rec id_caller'1 = function v_X -> id'1 v_X

let rec id_rec'1 = function v_X -> id_rec'1 v_X
